### ISSUE TITLE
It can't works well with react

### ISSUE THREAD

**Issue by leozim** [Lonefy/vscode-JS-CSS-HTML-formatter#37]
Like this line should be like this
and It became like this

<
View style = { styles.container } >

which cause error.

how to avoid that?

**Comment by zacklideveloper** [comment]
Yeah, I love this extension but can not use it with react especially JSX.
Will you guys try to fix it?

**Comment by theiwaz** [comment]
Same here.. would love for it to have an option to enable JSX formatting in js code instead of mangling my jsx templates. For now I've just turned off formatting on save.
